solarforecastarbiter/metrics/tests/test_probabilistic.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+import numpy as np
+from solarforecastarbiter.metrics import probabilistic as prob
+
+
+@pytest.mark.parametrize("fx,fx_prob,obs,value", [
+    # forecast 10 MW with 100% probability
+    (10, 100, 8, 0.0),   # actual: 8 MW
+    (10, 100, 10, 0.0),  # actual: 10 MW
+    (10, 100, 17, 1.0),  # actual: 17 MW
+
+    # forecast 8 MW with 50% probability
+    (8, 50, 2, 0.25),   # actual: 2 MW
+    (8, 50, 8, 0.25),   # actual: 8 MW
+    (8, 50, 13, 0.25),  # actual: 13 MW
+
+    # forecast 350 W/m^2 with 30% probability
+    (350, 30, 270, (0.3 - 1) ** 2),  # actual: 270 W/m^2
+    (350, 30, 350, (0.3 - 1) ** 2),  # actual: 350 W/m^2
+    (350, 30, 521, (0.3 - 0) ** 2),  # actual: 521 W/m^2
+
+    # forecast 2 kWh with 70% probability
+    (2, 70, 1, (0.7 - 1) ** 2),  # actual: 1 kWh
+    (2, 70, 2, (0.7 - 1) ** 2),  # actual: 2 kWh
+    (2, 70, 4, (0.7 - 0) ** 2),  # actual: 4 kWh
+
+    # vector inputs
+    (np.asarray([10, 10]), np.asarray([100, 100]), np.asarray([8, 8]), 0.0),
```

Comment:
`array` instead of `asarray` will make this marginally easier to read

Reply:
Done.